src/components/BookCollection/BookCollection.test.tsx

```diff
@@ -38,7 +38,7 @@ test("should not render load more button when number of showed books is equal to
   const books = BooksFixture;
```

Comment:
You can destructure books from fixture like this: const { books } = BooksFixture